change/@microsoft-teams-js-88216a56-3ec5-42b0-b7a5-db37b54879df.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added `disconnectConnector` to `{externalAppAuthentication}` capability that will disconnect the external app connector. The capability is still awaiting support in one or most host applications. To track availability of this capability across different hosts see https://aka.ms/capmatrix",
```

nit: should this be
```suggestion
  "comment": "Added `disconnectConnector` to `{externalAppAuthentication}` capability that will disconnect the external app connector. The capability is still awaiting support in one or more host applications. To track availability of this capability across different hosts see https://aka.ms/capmatrix",
```
And also do we publish the externalApp* capabilities to the support matrix?